internal/worker/worker.go

```diff
@@ -66,6 +68,13 @@ func New(opts ...Option) (*Worker, error) {
 		return nil, fmt.Errorf("%w: no upstreams were specified", ErrInitializationFailed)
 	}
 
+	// Image-related metrics
+	imagesCounter, err := meter.Int64Counter("org.cirruslabs.persistent_worker.images.total")
+	if err != nil {
+		panic(err)
```

Comment:
Why panic?

Reply:
Fixed in https://github.com/cirruslabs/cirrus-cli/pull/679/commits/9d8c6f265c688ca2d7029ea1607b43b33e60e546.